opm/simulators/wells/GroupStateHelper.cpp

```diff
@@ -385,8 +385,8 @@ GroupStateHelper<Scalar, IndexTraits>::checkGroupConstraintsProd(const std::stri
     for (std::size_t ii = 1; ii < num_ancestors; ++ii) {
         const int num_gr_ctrl = this->groupControlledWells(chain[ii],
                                                            /*always_included_child=*/"",
-                                                           /*is_producer=*/true,
-                                                           /*injection_phase_not_used=*/Phase::OIL);
+                                                           /*is_production_group=*/true,
```

Comment:
Why do we want to change `is_producer` to `is_production_group` here? I think both are readable, so no strong opinion.